src/components/TableAuditLog.vue

```diff
@@ -211,7 +211,7 @@ export default {
           userEmail: params.filter.userEmail,
           sortDirection: params.pagination.descending ? 'DESC' : 'ASC',
           offset: (params.pagination.page - 1) * params.pagination.rowsPerPage,
-          rowsPerPage: params.pagination.rowsPerPage
+          rowsPerPage: params.pagination.rowsPerPage === 0 ? this.pagination.rowsNumber : params.pagination.rowsPerPage
         }
```

Comment:
shouldn't it send undefined when rowsPerPage is 0?

Reply:
finito